File: leetcode/3516. 找到最近的人.go
func findClosest(x int, y int, z int) int {
	if math.Abs(float64(x-z)) == math.Abs(float64(y-z)) {
		return 0
	} else {
		if math.Abs(float64(x-z)) > math.Abs(float64(y-z)) {
			return 2
		} else {
			return 1
		}
	}
}

File: leetcode/3516. 找到最近的人.rs
impl Solution {
    pub fn find_closest(x: i32, y: i32, z: i32) -> i32 {
        if ((x - z).abs() == (y - z).abs()) {
            return 0;
        } else {
            if ((x - z).abs() > (y - z).abs()) {
                return 2;
            } else {
                return 1;
            }
        }
    }
}